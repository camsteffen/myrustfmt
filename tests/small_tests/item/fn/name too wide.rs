// test-kind: no-change

fn hiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii() {
    x;
}
