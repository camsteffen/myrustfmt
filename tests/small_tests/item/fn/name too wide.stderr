Max width exceeded at line 0
